### ISSUE TITLE
players can't see each other on the pioneer but in lobby they can

### ISSUE THREAD

**Issue by sirdankz** [fuzziqersoftware/newserv#15]
In the "main lobby" players can interact w/ each other perfectly but once in a "quest lobby" players cannot see one another, nor can they start a quest successfully. 

1.I've built and installed "phosg" from inside the folder made from phosg-master.zip
2. I ran cmake . && make from inside the folder made from newserv-master.zip
3. I've configured config.json to my internal and ex IP
4. I copied over the "System" folder and the file  "./newserv" to the "phosg-master" folder
5. I then ran "sudo ./newserv" in terminal.
6. I connected players successfully to my server
7. I also made sure all required ports were forwarded/opened

**Comment by fuzziqersoftware** [comment]
Sorry for the delay; I didn't have a setup for testing this until very recently.

I fixed several bugs today, including this one and another bug that caused very few item drops to occur in most games. Things should work better if you build a new version of the server from the master branch. (The code depends on some new stuff in phosg, so you'll have to rebuild + reinstall that too, unfortunately.)

**Comment by sirdankz** [comment]
No worries, you're the only person coding this publicly so us getting anything is good.

I forgot to mention some glitches..

1. players can only connect to each other by the host of the lobby using the "$maxlevel" command otherwise the game sends a message saying "player level to high to join"
2. players can now see each other at quest counter but once selecting a quest to download, the host softlocks and the guest never gets the downloading screen change

This could maybe be a result of me building the servers wrong? 

I first built (psosg) then I copied over the folders and files from inside of (masternewserv.zip) over to the (psosg) folder.

I then ran "cmake . && make" inside of the (psosg) folder containing the files built from
